src/io/sb3/fromSb3.ts

```diff
@@ -237,18 +243,23 @@ function getBlockScript(blocks: { [key: string]: sb3.Block }) {
       }
 
       if (block.opcode === OpCode.procedures_call) {
+        const mutation = (block as sb3.Block<OpCode.procedures_call>).mutation;
         result = {
-          PROCCODE: { type: "string", value: block.mutation.proccode },
+          PROCCODE: { type: "string", value: mutation.proccode },
           INPUTS: {
             type: "customBlockInputValues",
-            value: (JSON.parse(block.mutation.argumentids as string) as string[]).map(argumentid => {
-              let value = result[argumentid];
+            // TODO: Scratch itself uses the argumentids of the corresponding procedures_prototype block.
+            // There may be some cases where they go out of sync!
```

Comment:
I think the comment is misleading. Essentially, it's possible for a `procedures_call` block's arguments to desync with the corresponding `procedures_prototype`. This can happen if you drag a `procedures_call` block into the backpack, change the procedure definition around, then drag the block back out of the backpack.

This is something that the exporters should worry about, not the importers, so I'll remove this comment.